frontend/src/types.ts

```diff
@@ -106,3 +106,25 @@ export type BuildStatus = {
   status: BUILD_PHASE;
   timestamp: string;
 };
+
+export enum NotebookStatus {
+  VALIDITING,
```

Comment:
Typo: `s/VALIDITING/VALIDATING/`, missing `IMPORTING` status. Do we need enum in here? Why can't we consume the string value from backend directly?

Reply:
I like that idea better.  I'll make that change.